assets/src/blocks/Cookies/CookiesBlock.js

```diff
@@ -24,27 +26,27 @@ export class CookiesBlock {
         },
         necessary_cookies_name: {
           type: 'string',
-          default: ''
+          default: DEFAULT_COOKIES_COPY.necessary_cookies_label
```

Comment:
This default value is only used inside the editor, it's not saved or used anywhere else. The [back end still has different default values](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/c0d7ca2d7e01d895656f476bdaeba3fc15e294a1/classes/blocks/class-cookies.php#L32-L63).

Reply:
Updated! 🙂